packages/phoenix-evals/src/phoenix/evals/span_templates.py

```diff
@@ -186,6 +186,86 @@
 
 QA_SPAN_PROMPT_RAILS_MAP = OrderedDict({True: "correct", False: "incorrect"})
 
+FUNCTION_CALLING_SPAN_BASE_TEMPLATE = """
```

Comment:
Tool calling is the more up-to-date term if we are going by the OpenAI API. The function call parameters are deprecated. https://platform.openai.com/docs/api-reference/chat/create#chat-create-function_call